pixeltable/iterators/document.py

```diff
@@ -472,6 +480,67 @@ def _txt_sections(self) -> Iterator[DocumentSection]:
         assert self._doc_handle.txt_doc is not None
         yield DocumentSection(text=ftfy.fix_text(self._doc_handle.txt_doc), metadata=DocumentSectionMetadata())
 
+    def _markitdown_sections(self) -> Iterator[DocumentSection]:
```

Comment:
This looks like pure slop to me. Why does this need to be a separate method from `_markdown_sections()`? Either reuse `_markdown_sections()` and parameterize as necessary to minimize code duplication, or add a comment to explain the reasoning why this needs to be separate.